kratos/python/add_modeler_to_python.cpp

```diff
@@ -121,6 +122,20 @@ void  AddModelerToPython(pybind11::module& m)
     py::class_<CleanUpProblematicTrianglesModeler, CleanUpProblematicTrianglesModeler::Pointer, Modeler>(m, "CleanUpProblematicTrianglesModeler")
         .def(py::init<Model&, Parameters>())
     ;
+
+    py::class_<SurrogateBoundaryModeler::SurrogateBoundaryNode>(m, "SurrogateBoundaryNode")
+    .def("IsActive", &SurrogateBoundaryModeler::SurrogateBoundaryNode::IsActive)
+    .def("GetSignedDistance", &SurrogateBoundaryModeler::SurrogateBoundaryNode::GetSignedDistance)
+    .def("IsInside", &SurrogateBoundaryModeler::SurrogateBoundaryNode::IsInside)
+    .def("GetVectorDistance", &SurrogateBoundaryModeler::SurrogateBoundaryNode::GetVectorDistance, py::return_value_policy::reference_internal)
+    .def("GetNodePtr", &SurrogateBoundaryModeler::SurrogateBoundaryNode::GetNodePtr, py::return_value_policy::reference_internal);
+
+    py::class_<SurrogateBoundaryModeler, SurrogateBoundaryModeler::Pointer, Modeler>(m, "SurrogateBoundaryModeler")
+    .def(py::init<Model &, Parameters>())
+    .def("ComputeSurrogateBoundary", &SurrogateBoundaryModeler::ComputeSurrogateBoundary)
+    .def("GetSurrogateBoundaryNodes", &SurrogateBoundaryModeler::GetSurrogateBoundaryNodes)
+    .def("GetSurrogateBoundaryNode", &SurrogateBoundaryModeler::GetSurrogateBoundaryNode, py::return_value_policy::reference_internal)
+    .def("PrintSBData", &SurrogateBoundaryModeler::PrintSBData);
```

Comment:
I think it would be good that we stick to the standard `Modeler` API. Hence, I wouldn't expose all these these and to "our stuff" at the end of the `SetupModelPart` function of the API.

Reply:
Which is already exposed by the base class by the way.